### DISCUSSION TITLE
How strong is DDOS protection in hobby tariff?

### DISCUSSION THREAD

**Discussion by dikiyp** [vercel/community#4149] [Feature Question] [answered]
### Question

I have a question how strong is the protection on the hobby tariff and can you give me an example?

**Comment by swarnava** [comment, marked as answer]
Hobby plans are meant for experimental purpose / testing & understanding the platform platform and we don't provide uptime guarantee on Hobby plans. You shouldn't host production website which expects traffic on Hobby account. Although we have basic DDoS protection on Hobby plan.

Learn more about our Fair Usage Policy here: https://vercel.com/docs/concepts/limits/fair-use-policy
